stream-chat-android-offline/build.gradle

```diff
@@ -70,13 +70,13 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
         freeCompilerArgs += [
                 '-progressive',
                 '-Xexplicit-api=strict',
-                '-Xopt-in=io.getstream.chat.core.internal.InternalStreamChatApi',
+                '-Xopt-in=io.getstream.chat.android.core.internal.InternalStreamChatApi',
         ]
     }
 }
 
 dependencies {
-    implementation project(':stream-chat-android-core')
+    api project(':stream-chat-android-core')
```

Comment:
`stream-chat-android-client` already provide `stream-chat-android-core` as api dependency, do we need to provide it from here? :thinking:

Reply:
Looks like we don't need to be explicit about it, removed these.